rails_event_store/lib/rails_event_store/async_handler_helpers.rb

```diff
@@ -2,21 +2,32 @@
 
 module RailsEventStore
   module AsyncHandler
-    def self.with_defaults
-      with
-    end
+    class << self
```

Comment:
I don't think this was necessary change. Changing formatting made it harder to notice what really changed.